### PR TITLE
Grid Layout talk proposal 🐈

### PR DESCRIPTION
**Pull request by pepelsbey**

### CODE REVIEW

File: content/events/moscowjs-30.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+## Можно вообще всё. Раскладка по гриду
+
+Верстаете как юный бог на флексах или пасёте флоуты по углам — неважно. Когда-нибудь вам придётся перейти на грид-раскладки. Не просто инлайн-блоки на стероидах, как флексы, а принципиально новую систему, созданную специально для сложных раскладок. Понятное введение и примеры использования для реальных задач.
+
+### Вадим Макеев, Opera
+
+![Вадим Макеев](/images/speakers/pepelsbey.jpg)
+
+Twitter: [@pepelsbey](https://twitter.com/pepelsbey)
+GitHub: [@pepelsbey](https://twitter.com/pepelsbey)
```
Comment (vslinko) [inline content/events/moscowjs-30.md]: wrong url

File: content/events/moscowjs-30.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# MoscowJS 29
```
Comment (vslinko) [inline content/events/moscowjs-30.md]: moscowjs 30

File: content/events/moscowjs-30.md (hunk 1 above)
Comment (pepelsbey) [inline content/events/moscowjs-30.md]: fixed

File: content/events/moscowjs-30.md (hunk 2 above)
Comment (pepelsbey) [inline content/events/moscowjs-30.md]: fixed

**Pull request merged**